Makefile

```diff
@@ -28,6 +28,32 @@ all: $(RUNC_LINK)
 static: $(RUNC_LINK)
 	CGO_ENABLED=1 go build -i -tags "$(BUILDTAGS) cgo static_build" -ldflags "-w -extldflags -static -X main.gitCommit=${COMMIT} -X main.version=${VERSION}" -o runc .
 
+release: $(RUNC_LINK)
+	@flag_list=(seccomp selinux apparmor static); \
```

Comment:
This looks quite nice IMO. But maybe we should move this to an actual `.sh` script in `script/...` and call it from the Makefile (so we can avoid the `; \`, `$$` and other ugliness)?